app/models/project.rb

```diff
@@ -131,4 +130,32 @@ def project_events(order_by = "Completed IS NULL OR Completed = '0000-00-00', Co
   def health_status_available?
     return false
   end
+
+  def self.new_order(person)
+    "case when projects.primary_agent_id = #{person.id} and projects.status_value = 'active'
+          and projects.type = 'Loan' then 8
+
+          when projects.primary_agent_id = #{person.id} and projects.status_value = 'active'
+          and projects.type = 'BasicProject' then 7
+
+          when projects.primary_agent_id = #{person.id} and projects.status_value = 'prospective'
+          and projects.type = 'Loan' then 6
+
+          when projects.primary_agent_id = #{person.id} and projects.status_value = 'prospective'
+          and projects.type = 'BasicProject' then 5
+
+          when projects.secondary_agent_id = #{person.id} and projects.status_value = 'active'
+          and projects.type = 'Loan' then 4
+
+          when projects.secondary_agent_id = #{person.id} and projects.status_value = 'active'
+          and projects.type = 'BasicProject' then 3
+
+          when projects.secondary_agent_id = #{person.id} and projects.status_value = 'prospective'
+          and projects.type = 'Loan' then 2
+
+          when projects.secondary_agent_id = #{person.id} and projects.status_value = 'prospective'
+          and projects.type = 'BasicProject' then 1
+
```

Comment:
Next in the list is projects where the user is the secondary agent and anything with one of the OPEN_STATUES not 'prospective' and 'active'. This is sorted first by Loan and then by Basic Project.

The next two items are:
All remaining loans in the list
All remaining basic projects in the list

The last filter can, of course, be the else below.